apps/ui/src/components/EditorVotingType.vue

```diff
@@ -4,28 +4,33 @@ import { Draft, VoteType } from '@/types';
 import voteBasicIllustration from '@/assets/images/vote-basic.svg';
 import voteSingleChoiceIllustration from '@/assets/images/vote-single-choice.svg';
 import voteApprovalIllustration from '@/assets/images/vote-approval.svg';
+import voteBasicDarkIllustration from '@/assets/images/vote-basic-dark.svg';
+import voteSingleChoiceDarkIllustration from '@/assets/images/vote-single-choice-dark.svg';
+import voteApprovalDarkIllustration from '@/assets/images/vote-approval-dark.svg';
 
 const proposal = defineModel<Draft>({ required: true });
 
 defineProps<{
   votingTypes: VoteType[];
 }>();
 
+const { getMode } = useUserSkin();
+
 const VOTING_TYPES_INFO = {
   basic: {
     label: 'Basic voting',
     description: 'Single choice voting with three choices: For, Against or Abstain.',
-    image: voteBasicIllustration
+    image: getMode() === 'dark' ? voteBasicDarkIllustration : voteBasicIllustration
```

Comment:
It's not ideal, because it's not reactive. Not really an issue now (there is no visible button to change theme), but not a good thing to depend on. We can make it reactive and deprecate a `getMode` entirely and replace it with `currentMode`:
```diff
diff --git a/apps/ui/src/components/App/Topnav.vue b/apps/ui/src/components/App/Topnav.vue
index 876ee15e..5a2dba8d 100644
--- a/apps/ui/src/components/App/Topnav.vue
+++ b/apps/ui/src/components/App/Topnav.vue
@@ -8,7 +8,7 @@ const auth = getInstance();
 const uiStore = useUiStore();
 const { modalAccountOpen } = useModal();
 const { login, web3 } = useWeb3();
-const { toggleSkin, getMode } = useUserSkin();
+const { toggleSkin, currentMode } = useUserSkin();
 
 const loading = ref(false);
 const searchInput = ref();
@@ -97,7 +97,7 @@ watch(route, to => {
         </UiButton>
         <IndicatorPendingTransactions class="ml-2" />
         <UiButton class="!px-0 w-[46px] ml-2" @click="toggleSkin">
-          <IH-light-bulb v-if="getMode() === 'dark'" class="inline-block" />
+          <IH-light-bulb v-if="currentMode === 'dark'" class="inline-block" />
           <IH-moon v-else class="inline-block" />
         </UiButton>
       </div>
diff --git a/apps/ui/src/components/EditorVotingType.vue b/apps/ui/src/components/EditorVotingType.vue
index 4f4c6893..3447686f 100644
--- a/apps/ui/src/components/EditorVotingType.vue
+++ b/apps/ui/src/components/EditorVotingType.vue
@@ -14,25 +14,26 @@ defineProps<{
   votingTypes: VoteType[];
 }>();
 
-const { getMode } = useUserSkin();
+const { currentMode } = useUserSkin();
 
-const VOTING_TYPES_INFO = {
+const VOTING_TYPES_INFO = computed(() => ({
   basic: {
     label: 'Basic voting',
     description: 'Single choice voting with three choices: For, Against or Abstain.',
-    image: getMode() === 'dark' ? voteBasicDarkIllustration : voteBasicIllustration
+    image: currentMode.value === 'dark' ? voteBasicDarkIllustration : voteBasicIllustration
   },
   'single-choice': {
     label: 'Single choice voting',
     description: 'Each voter may select only one choice.',
-    image: getMode() === 'dark' ? voteSingleChoiceDarkIllustration : voteSingleChoiceIllustration
+    image:
+      currentMode.value === 'dark' ? voteSingleChoiceDarkIllustration : voteSingleChoiceIllustration
   },
   approval: {
     label: 'Approval voting',
     description: 'Each voter may select any number of choices.',
-    image: getMode() === 'dark' ? voteApprovalDarkIllustration : voteApprovalIllustration
+    image: currentMode.value === 'dark' ? voteApprovalDarkIllustration : voteApprovalIllustration
   }
-};
+}));
 
 function handleVoteTypeSelected(type: VoteType) {
   if (!proposal.value) return;
diff --git a/apps/ui/src/composables/useUserSkin.ts b/apps/ui/src/composables/useUserSkin.ts
index 7040aa00..dd479bf0 100644
--- a/apps/ui/src/composables/useUserSkin.ts
+++ b/apps/ui/src/composables/useUserSkin.ts
@@ -9,7 +9,7 @@ const currenSkin = lsGet('skin', NOT_SET);
 const osSkin = DARK_MODE;
 
 const userSkin = ref(currenSkin === NOT_SET ? osSkin : currenSkin);
-const getMode = () => (userSkin.value === LIGHT_MODE ? LIGHT_MODE : DARK_MODE);
+const currentMode = computed(() => (userSkin.value === LIGHT_MODE ? LIGHT_MODE : DARK_MODE));
 const _toggleSkin = skin => {
   if (skin === LIGHT_MODE) {
     lsSet('skin', DARK_MODE);
@@ -44,7 +44,7 @@ export function useUserSkin() {
 
   return {
     userSkin,
-    getMode,
+    currentMode,
     toggleSkin
   };
 }
diff --git a/apps/ui/src/views/Editor.vue b/apps/ui/src/views/Editor.vue
index 2d885eeb..39c2850b 100644
--- a/apps/ui/src/views/Editor.vue
+++ b/apps/ui/src/views/Editor.vue
@@ -271,40 +271,6 @@ export default defineComponent({
 
 <template>
   <div>
-    <nav class="border-b bg-skin-bg fixed top-0 z-50 right-0 left-0 lg:left-[72px]">
-      <div class="flex items-center h-[71px] mx-4">
-        <div class="flex-auto space-x-2">
-          <router-link :to="{ name: 'space-overview', params: { id: param } }" class="mr-2">
-            <UiButton class="leading-3 w-[46px] !px-0">
-              <IH-arrow-narrow-left class="inline-block" />
-            </UiButton>
-          </router-link>
-          <h4 class="py-2 inline-block">New proposal</h4>
-        </div>
-        <IndicatorPendingTransactions class="mr-2" />
-        <UiLoading v-if="!space" class="block p-4" />
-        <div v-else class="space-x-2">
-          <UiButton
-            class="float-left leading-3 !pl-3 !pr-2.5 rounded-r-none"
-            @click="modalOpen = true"
-          >
-            <IH-collection class="inline-block" />
-          </UiButton>
-          <UiButton
-            class="rounded-l-none border-l-0 float-left !m-0 !px-3"
-            :loading="sending || (web3.account !== '' && fetchingVotingPower)"
-            :disabled="!canSubmit"
-            @click="handleProposeClick"
-          >
-            <span
-              class="hidden mr-2 md:inline-block"
-              v-text="proposal?.proposalId ? 'Update' : 'Publish'"
-            />
-            <IH-paper-airplane class="inline-block rotate-90" />
-          </UiButton>
-        </div>
-      </div>
-    </nav>
     <UiContainer v-if="proposal" class="pt-5 !max-w-[660px] mx-0 md:mx-auto s-box">
       <UiAlert v-if="!fetchingVotingPower && !votingPowerValid" type="error" class="mb-4">
         You do not have enough voting power to create proposal in this space.
```